server/routes/amendAReferral/changeLogController.test.ts

```diff
@@ -113,6 +114,14 @@ describe('GET /referrals/:referralId/changelog', () => {
   beforeEach(() => {
     interventionsService.getSentReferral.mockResolvedValue(referral)
     interventionsService.getChangelog.mockResolvedValue(changelog)
+    interventionsService.getIntervention.mockResolvedValue(
+      interventionFactory.build({
+        contractType: {
+          code: 'accomdation',
```

Comment:
```suggestion
          code: 'accommodation',
```
As per @awhodgson256's suggestion above

Reply:
Thank you for pointing out, it was a typo. I will correct it